src/wp-includes/l10n.php

```diff
@@ -833,11 +833,13 @@ function load_textdomain( $domain, $mofile, $locale = null ) {
 		 * the file path could be for an MO or PHP file.
 		 *
 		 * @since 6.5.0
+		 * @since x.y.z The `$locale` parameter was added.
```

Comment:
In line 761 we added `$locale`, for consistency update docblock.

```suggestion
		 * @since x.y.z Added the `$locale` parameter.
```

Reply:
FWIW we use both formats interchangeably in core